insights/specs/default.py

```diff
@@ -726,7 +726,7 @@ def pcp_enabled(broker):
     @datasource(Sap, HostContext)
     def sap_sid(broker):
         sap = broker[Sap]
```

Comment:
Please add a docstring to this datasource.

Reply:
Done